numpy/lib/nanfunctions.py

```diff
@@ -40,6 +40,30 @@
     ]
 
 
+def _nan_mask(a, out=None):
+    """
+    Parameters
+    ----------
+    a : array-like
+        Input array with at least 1 dimension.
+    out : ndarray, optional
+        Alternate output array in which to place the result.  The default
+        is ``None``; if provided, it must have the same shape as the
+        expected output and will prevent the allocation of a new array.
+
+    Returns
+    -------
+    y : ndarray
+        A bool array where ``np.nan`` positions are marked with ``False``
+        and other positions are marked with ``True``.
+    """
+    a = np.asarray(a)
+    if a.ndim < 1:
+        raise ValueError("Input array should be >= 1D")
```

Comment:
Is there any reason to require this?

Reply:
Good point -- full test suite passes locally when I remove it so revised accordingly